### DISCUSSION TITLE
Q: Code execution is not possible for Claude, correct?

### DISCUSSION THREAD

**Discussion by TimonGisler** [danny-avila/LibreChat#4560] [Q&A] [answered]
Claude.ai and ChatGPT.com both allow for code execution in their UI; for example, I can ask Claude to write regex to filter some data, and it writes the regex plus some code to test it.

I am currently searching for a local web UI that allows me to do that.

The artifact kind of works, but Claude writes an entire UI, which is unnecessary and wastes tokens.

There seem to exist assistants, but if I understood correctly, this only works for OpenAI: https://www.librechat.ai/docs/configuration/pre_configured_ai/assistants

In the README, I also found "Non-OpenAI Agents in Active Development 🚧," which I think means Claude (and other models) will soonish get a Code Interpreter.

I am asking because this UI looks amazing, and if this feature is already implemented and just a little bit hidden, I would love to use it.

I made an search for similar Questions but only found:
1. https://github.com/danny-avila/LibreChat/discussions/4524 --> no answers

**Comment by danny-avila** [comment, marked as answer]
It's in the works and will release by early November latest
